tensorboard/uploader/uploader_main.py

```diff
@@ -363,8 +374,16 @@ def execute(self, server_info, channel):
                 ("Runs", str(experiment.num_runs)),
                 ("Tags", str(experiment.num_tags)),
             ]
-            for (name, value) in data:
-                print("\t%s %s" % (name.ljust(12), value))
+            if self.json:
+                experiments_json.append(
+                    collections.OrderedDict([("URL", url)] + data)
```

Comment:
Could we just pass `sort_keys=True` instead of using an `OrderedDict`?
The determinism is nice, but I don’t know that we really need to enforce
the same order as with the human-readable output; sorting keys is the
standard “stable stringify”.